README.md

```diff
@@ -172,7 +172,7 @@ The upcoming features include:
 
 ## Supported OceanBase Versions
 
-ob-operator supports OceanBase v4.x versions. The validated versions include 4.1.x and 4.2.x. It will continue to support new versions of the OceanBase community edition.
+ob-operator supports OceanBase v4.x versions. The validated versions include 4.1.x, 4.2.x and 4.3.x. It will continue to support new versions of the OceanBase community edition.
```

Comment:
we can drop 'The validated versions include 4.1.x, 4.2.x and 4.3.x' and don't need to update it for every OceanBase release, maybe change it to 'Some features may require a specific OceanBase version. Please consult the documentation for details.'